server/v2/tests/auth.spec.js

```diff
@@ -0,0 +1,51 @@
+import Chai from "chai";
+import chaiHttp from "chai-http";
+
+import app from "../../app";
+import usersTester from "./mockData/authData";
+
+const { expect } = Chai;
+Chai.use(chaiHttp);
+
+describe('Endpoint /api/v2/auth/signup', () => {
+
+    it("should create a new user account",
+        async() => {
+            const res = await Chai.request(app)
+                .post("/api/v2/auth/signup")
+                .send(usersTester[0]);
+            expect(res.body.status).to.equal(201);
+            expect(res.body).to.be.an('object');
+            expect(res.body.message).to.be.a('string');
+        });
```

Comment:
I'm glad that you are using async-await in your tests. Usually, when you are using async-await you need to nest the block inside a try-catch block to capture the errors.  that's the only thing which is missing in your test.

Reply:
OK, I'm going to work on it according to the resource you gave us yesterday. Thanks again